third_party/conan/configs/linux/profiles/coverage_clang9_relwithdebinfo

```diff
@@ -0,0 +1,24 @@
+C_COMPILER=clang-9
+CXX_COMPILER=clang++-9
+C_FLAGS= -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fsized-deallocation -D_FORTIFY_SOURCE=2 -fstack-protector-all -fprofile-instr-generate -fcoverage-mapping
+CXX_FLAGS= -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fsized-deallocation -D_FORTIFY_SOURCE=2 -fstack-protector-all -fprofile-instr-generate -fcoverage-mapping
+LINKER_FLAGS= -Wl,-z,relro,-z,now,-z,noexecstack
+[settings]
+os=Linux
+os_build=Linux
+arch=x86_64
+arch_build=x86_64
+compiler=clang
+compiler.version=9
+compiler.libcxx=libstdc++11
+compiler.fpo=False
+build_type=RelWithDebInfo
+[options]
+[build_requires]
+&: cmake_installer/3.16.3@conan/stable
+[env]
+CC=$C_COMPILER
+CXX=$CXX_COMPILER
+CFLAGS=$C_FLAGS
+CXXFLAGS=$CXX_FLAGS
+LDFLAGS=$LINKER_FLAGS
```

Comment:
The profiles are written to be easily extended by other profile. Maybe it makes sense to write an extending profile, something like this:

```
include(clang9_relwithdebinfo)

[env]
CFLAGS=$CFLAGS -fcoverage-mapping
CXXFLAGS=$CXXFLAGS -fcoverage-mapping

Reply:
I did not know this was possible, neat!